system-int-test/test/cmr/system_int_test/search/service/service_search_test.clj

```diff
@@ -0,0 +1,109 @@
+(ns cmr.system-int-test.search.service.service-search-test
+  "This tests searching services."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :refer [are3]]
+   [cmr.mock-echo.client.echo-util :as e]
+   [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.association-util :as au]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.search-util :as search]
+   [cmr.system-int-test.utils.service-util :as services]))
+
+(use-fixtures :each
+              (join-fixtures
+               [(ingest/reset-fixture {"provguid1" "PROV1" "provguid2" "PROV2"})
+                services/grant-all-service-fixture]))
+
+(deftest search-for-services-validation-test
+  (testing "Unrecognized parameters"
+    (is (= {:status 400, :errors ["Parameter [foo] was not recognized."]}
+           (search/find-refs :service {:foo "bar"}))))
+
+  (testing "Unsupported sort-key parameters"
```

Comment:
Do we have any supported sort keys? If so we should add a test and add to the documentation.

Reply:
We don't yet. It is my plan to file an issue and work on it next.